.githooks/pre-commit

```diff
@@ -3,6 +3,21 @@ cd "$(git rev-parse --show-toplevel)"
 ESLINT="client/node_modules/.bin/eslint"
 pwd
 
+staged=`git diff --name-only --cached --diff-filter=A --diff-filter=M | cat`
+pylint="c:\users\micky\appdata\local\programs\python\python310\lib\site-packages  --rcfile .pylintrc --load-plugins pylint_django --load-plugins pylint_quotes --reports=no"
+
+if [ "$staged" ]
```

Comment:
I think we should also have an `IF PYLINT` along with the error message if not found

Reply:
Would having something similar to lines 21-24 work? like instead of eslint, it's the error for pylint instead